hc/accounts/tests/test_check_token.py

```diff
@@ -10,19 +10,27 @@ def setUp(self):
         self.profile.save()
 
     def test_it_shows_form(self):
-        r = self.client.get("/accounts/check_token/alice/secret-token/")
-        self.assertContains(r, "You are about to log in")
+        response = self.client.get("/accounts/check_token/alice/secret-token/")
+        self.assertContains(response, "You are about to log in")
 
     def test_it_redirects(self):
-        r = self.client.post("/accounts/check_token/alice/secret-token/")
-        self.assertRedirects(r, "/checks/")
+        response = self.client.post("/accounts/check_token/alice/secret-token/")
+        self.assertRedirects(response, "/checks/")
 
         # After login, token should be blank
         self.profile.refresh_from_db()
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_if_already_logged_in(self):
+        self.client.login(username="alice@example.org", password="password")
+        response = self.client.get("/accounts/check_token/alice/secret-token/")
+        assert response.status_code == 302
```

Comment:
maybe some consistency with the tests. If you are asserting the status code maybe assert for all